Ryujinx/Ui/GLRenderer.cs

```diff
@@ -486,8 +486,8 @@ private bool UpdateFrame()
 
                     TouchPoint currentPoint = new TouchPoint
                     {
-                        X = mX,
-                        Y = mY,
+                        X = (uint)mX,   // Can't be -ve?
```

Comment:
```suggestion
                        X = (uint)mX, // Can't be -ve?
```
What does that mean?